accessibility/accessibility-checklist.md

```diff
@@ -46,25 +46,7 @@ Not all users are able to easily navigate by landmarks. Multiple navigation link
 
 To help them out you can add a "Skip Navigation" link to the top of each page which, when activated, will take them to the main section of the page.
 
-We do this:
-
-```
-<body>
-<p><a href="#maincontent">Skip to main content</a></p>
-...
-<main id="maincontent" tabindex="-1">
-<h1>Heading</h1>
-<p>This is the first paragraph</p>
-```
-
-We add `tabindex="-1"` to the targeted element so that it can be programmatically focussed when skipped to.
-
-You should supply CSS that hides the Skip Navigation _for sighted users only_, and makes the link highly visible when it receives focus via tabbing.
-
-Resources:
-
-* [Article: WebAIM's overview of Skip Navigation techniques](https://webaim.org/techniques/skipnav/)
```

Comment:
would it be beneficial to keep this link in for 'further reading'?

Reply:
Yeah, that's a good idea. Got a bit delete key-happy :D